src/Miningcore/Payments/PaymentSchemes/PPSPaymentScheme.cs

```diff
@@ -45,9 +56,19 @@ public class PPSPaymentScheme : IPayoutScheme
         private readonly IBlockRepository blockRepo;
         private readonly IConnectionFactory cf;
         private readonly IShareRepository shareRepo;
+        private readonly IStatsRepository statsRepo;
+        private readonly IMapper mapper;
         private static readonly ILogger logger = LogManager.GetLogger("PPS Payment", typeof(PPSPaymentScheme));
+        private static IMemoryCache cache = new MemoryCache(new MemoryCacheOptions());
```

Comment:
May I know where are we configured 24hrs cache expiry?

Reply:
While setting the cache in GetBlockReward() method, I am setting the expiry to 24 hrs.

Reply:
Generally, we set expiry in MemoryCacheOptions since it's used only to store rewards. But it's fine this time to set it during cache value setup.